src/main/webapp/basic-search.js

```diff
@@ -168,26 +154,16 @@ export const populateDefaultQuery = (
 const MatchTypes = ({ state = [], setState, errors = {} }) => {
   errors = errors.matchTypesErrors || {}
   return (
-    <FormControl fullWidth>
-      <InputLabel>Match Types</InputLabel>
-      <Select
-        error={errors.datatypes !== undefined}
-        multiple
-        value={state}
-        onChange={e => setState(e.target.value)}
-        renderValue={selected => selected.join(', ')}
-      >
-        {datatypes.map(datatype => (
-          <MenuItem key={datatype} value={datatype}>
-            <Checkbox checked={state.indexOf(datatype) > -1} />
-            <ListItemText primary={datatype} />
-          </MenuItem>
-        ))}
-      </Select>
-      <FormHelperText error={errors.datatypes !== undefined}>
-        {errors.datatypes}
-      </FormHelperText>
-    </FormControl>
+    <FacetedDropdown
+      label="Match Types"
+      facetAttribute="datatype"
```

Comment:
Need to grab basicSearchMatchType from system properties instead of hard coding datatype